.cmake-format.py

```diff
@@ -1,3 +1,7 @@
+# Copyright (c) 2024 Orange. All rights reserved.
```

Comment:
s/bannor/banner/ in commit message for https://github.com/KhiopsML/khiops/pull/363/commits/5cdbba9b0097166236412081701004e7685ed22c